packages/ketcher-core/src/application/render/restruct/resimpleObject.ts

```diff
@@ -189,6 +191,14 @@ class ReSimpleObject extends ReObject {
     return refPoints;
   }
 
+  getHoverPathStyle(path: any, render: Render, type: boolean) {
+    if (type) {
+      return path.attr(render.options.hoverStyle);
+    } else {
+      return path.attr({ ...render.options.hoverStyle, fill: '#fff' });
+    }
+  }
```

Comment:
Not sure that "type" is a good name here. Could you please change it or describe what is it?